libs/core/src/lib/popover/popover-body/popover-body.component.scss

```diff
@@ -9,6 +9,11 @@ $fd-popover-border-radius: var(--fdPopover_Border_Radius);
     border-#{$position}-left-radius: $radius;
 }
 
+fd-popover-body {
+    max-width: 100%;
```

Comment:
Should these be vh/vw rather than % if we're concerned with the viewport?

Reply:
this element is placed inside `cdk-overflay` material's element and the cdk is controlling resizing, max-height\width, we only want our element to be not bigger that its container.